File: common-core/src/main/java/com/ityu/dao/system/TaskRepository.java

package com.ityu.dao.system;




import com.ityu.bean.entity.system.Task;
import com.ityu.dao.BaseRepository;

import java.util.List;

public interface TaskRepository extends BaseRepository<Task,Long> {

    long countByNameLike(String name);

    List<Task> findByNameLike(String name);
    List<Task> findAllByDisabled(boolean disable);
}

File: common-core/src/main/java/com/ityu/common/dao/system/TaskRepository.kt
package com.ityu.common.dao.system

import com.ityu.common.bean.entity.system.Task
import com.ityu.common.dao.BaseRepository

interface TaskRepository : BaseRepository<Task?, Long?> {
    fun countByNameLike(name: String?): Long
    fun findByNameLike(name: String?): List<Task?>?
    fun findAllByDisabled(disable: Boolean): List<Task?>?
}
